### ISSUE TITLE
Manual collection management user experience

### ISSUE THREAD

**Issue by HunterZ** [xbmc/xbmc#15984]
### Describe the bug
Here is a clear and concise description of what the problem is:

There are a couple of annoying usability issues manually creating and adding movies to a collection.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

After adding a movie to a new collection, the collection should show up in the list of collections when trying to add the second movie.

Additionally, the movie browser should not jump back to the top of the list after a movie is added to a collection.

## Actual Behavior

After adding a movie to a new collection, the collection is NOT listed when trying to add the second movie. You have to add the second movie to a "new" collection with the same exact name, by typing it in. After two movies are in the collection, it will show up in both the movie list and the add-to-collection list.

Additionally, after adding a movie to a collection, the movie browser jumps back to the top of the movie list.

### To Reproduce

1. Add a movie to a new collection.
2. Attempt to add a second movie to the collection.
3. Wonder why people thought this was OK.

### Debuglog

The debuglog can be found here:
N/A

### Screenshots 
Here are some links or screenshots to help explain the problem:

N/A

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

N/A

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version:
18.1 release

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
> After adding a movie to a new collection, the collection should show up in the list of collections when trying to add the second movie.

See: https://github.com/xbmc/xbmc/issues/15814

Enable "Include sets containing a single movie" and it will show up.

> Additionally, the movie browser should not jump back to the top of the list after a movie is added to a collection.

Confirmed. Probably it could behave in the same way as if you remove an item from the library, where you aren't pointed to the top of the list but to the next item which comes after the move you removed.

This log shows how I first added a movie to a random set and then I have removed a movie from the library. At the first action I'm pointed to the top of the list, For the 2nd action, I'm pointed to the next item in the list.

http://paste.ubuntu.com/p/qtvwDmnNcQ/
